monitor/seaclouds-data-collector/src/main/java/eu/seaclouds/monitor/seacloudsDc/Registry.java

```diff
@@ -1,16 +1,21 @@
-package eu.seaclouds.monitor.nuroDc;
+package eu.seaclouds.monitor.seacloudsDc;
 
 import it.polimi.tower4clouds.data_collector_library.DCAgent;
 import it.polimi.tower4clouds.manager.api.ManagerAPI;
 
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
 import java.util.Properties;
 import java.util.Set;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import eu.seaclouds.monitor.nuroDc.metrics.NuroMetric;
+import eu.seaclouds.monitor.seacloudsDc.metrics.NuroMetric;
+import eu.seaclouds.monitor.seacloudsDc.metrics.ModuleAvailability;
 import it.polimi.tower4clouds.model.data_collectors.DCDescriptor;
 import it.polimi.tower4clouds.model.ontology.InternalComponent;
 import it.polimi.tower4clouds.model.ontology.Resource;
```

Comment:
Could you please add some javadoc to this `Registry` class, its purpose is not clear.